lightningd/chaintopology.h

```diff
@@ -116,6 +117,9 @@ struct chain_topology {
 	/* Force a particular fee rate regardless of estimatefee (satoshis/kw) */
 	u32 *dev_override_fee_rate;
 #endif
+
+	/* lightningd's status so we can update it when we're done syncing */
+	int *lightningd_status;
```

Comment:
yeah sorry this int was a screwup in my patch during fixups. I like the topology status idea, then perhaps the lightningd status could be computed instead of as an enum?

Reply:
perhaps something like:

```c
enum chaintopology_status {
	CHAINTOPOLOGY_INITIALIZING,
	CHAINTOPOLOGY_SYNCING,
	CHAINTOPOLOGY_AT_TIP,
};
```